.github/workflows/docker.yaml

```diff
@@ -17,11 +17,11 @@ jobs:
     strategy:
       matrix:
         include:
-        - name: "2.4.0_cu124"
-          base_image: mosaicml/pytorch:2.4.0_cu124-python3.11-ubuntu20.04
+        - name: "2.5.1_cu124"
```

Comment:
sorry, one more. for this pr you should just _add_ the new docker builds, not replace the old ones

Reply:
fixed